app/context/CartContext.js

```diff
@@ -113,6 +120,7 @@ function CartProvider({ children }) {
       value={{
         ...cart,
         addToCart,
+        cartDispatch,
```

Comment:
@ynnoj perhaps this could be exported as resetCart instead. The Context in my opinion should take care of invoking dispatch and return functions that dispatch internally.